pandas/tests/test_generic.py

```diff
@@ -1539,6 +1540,15 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_deepcopy_empty(self):
```

Comment:
@jreback sure, but that would be a separate test from the one you said to move.  That issue never affected series; as far as I can tell, there's no way that it could have since it depends on having columns.

I have no problem adding generic tests for copy/deepcopy, but I want to make sure it's the right test.